src/Raven.Server/Documents/Handlers/Processors/Batches/AbstractBatchHandlerProcessorForBulkDocs.cs

```diff
@@ -111,6 +111,11 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, RequestHandl
                 if (indexBatchOptions != null)
                     command.ModifiedCollections = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
 
+                if (HttpContext.Request.Query.TryGetValue("noreply", out var noReply))
+                {
+                    command.IncludeReply = !bool.Parse(noReply);
```

Comment:
```suggestion
                    command.IncludeReply = bool.Parse(noReply) == false;
```

That would be what by default?

Reply:
It is 'true' by default. I initially wrote it in the negative, but I continue making mistakes until I said: "Screw this" I will just go with the positive instead.

Reply:
avoid using `!` as Oren suggested